packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -545,6 +563,74 @@ class TransferRootsDb extends BaseDb {
     return filtered as ExitableTransferRoot[]
   }
 
+  async getRelayableTransferRoots (
+    filter: GetItemsFilter = {}
+  ): Promise<RelayableTransferRoots[]> {
+    await this.tilReady()
+    // TODO: Remove this post-nitro
+    if (!isNitroLive) {
+      return []
+    }
+    const transferRoots: TransferRoot[] = await this.getTransferRootsFromTwoWeeks()
+    const filtered = transferRoots.filter(item => {
+      if (!item.sourceChainId) {
+        return false
+      }
+
+      if (!this.isRouteOk(filter, item)) {
+        return false
+      }
+
+      if (!item.destinationChainId) {
+        return false
+      }
+
+      if (item.isNotFound) {
+        return false
+      }
+
+      const destinationChain = chainIdToSlug(item.destinationChainId)
+      if (!RelayableChains.includes(destinationChain)) {
+        return false
+      }
+
+      // TODO: Remove this one week post-nitro
+      if (item.bondedAt && item.bondedAt < nitroStartTimestamp) {
+        return false
+      }
+
+      // TODO: Remove this one week post-nitro
+      if (item.confirmedAt && item.confirmedAt < nitroStartTimestamp) {
+        return false
+      }
+
+      const isSeenOnL1 = item?.bonded ?? item?.confirmed
+
+      let sentTxTimestampOk = true
+      if (item.sentRelayTxAt) {
+        sentTxTimestampOk = item.sentRelayTxAt + TxRetryDelayMs < Date.now()
+      }
+
+      // bondedAt should be checked first because a root can have both but it should be bonded prior to being confirmed
+      const seenOnL1Timestamp = item?.bondedAt ?? item?.confirmedAt
```

Comment:
I think a check for this values should be done to return early if they don't exists

```ts
if (!(item?.bondedAt || item?.confirmedAt)) {
  return false
}
```

otherwise I think it's possible `seenOnL1Timestamp` can be `undefined`